oaknational/Oak-Web-Application
# Added "video" and "partner bio" to curriculum overview table of contents

## Description
Added "video" and "partner bio" to curriculum overview table of contents

<img width="1328" alt="Screenshot 2024-10-07 at 11 41 19" src="https://github.com/user-attachments/assets/45155a7d-ec7f-4e89-b1b9-a72ee256986b">

## Issue(s)

Fixes `CUR-939`

## How to test
Cycle 1

1. Go to https://deploy-preview-2855--oak-web-application.netlify.thenational.academy
2. Check for regressions

Cycle 2

1. Checkout locally with `ENABLE_CYCLE_2=true`
2. Check "Video Guide" and "Our curriculum partner" work as expected in all viewport sizes
